language.go

```diff
@@ -181,6 +181,7 @@ var Exts = map[string]string{
 	"f03":         "FORTRAN Modern",
 	"f08":         "FORTRAN Modern",
 	"gleam":       "Gleam",
+	"g4":          "ANTLR v4 Grammar",
```

Comment:
This part only defines the file extension.
Language definitions need to be added in the following places:
https://github.com/hhatto/gocloc/blob/master/language.go#L505

```go
// NewDefinedLanguages create DefinedLanguages.
func NewDefinedLanguages() *DefinedLanguages {
	return &DefinedLanguages{
		Langs: map[string]*Language{
			"ActionScript":        NewLanguage("ActionScript", []string{"//"}, [][]string{{"/*", "*/"}}),
```

Reply:
Oh! I will add this :)